overrides/config/CustomMainMenu/mainmenu.json

```diff
@@ -87,33 +87,26 @@
             }
         },
 
-        "changelog":
+        "update":
         {
-            "text" : "Changes",
+            "text" : "Update Notes",
             "posX" : 125,
-            "posY" : 2,
-            "width" : 120,
-            "height" : 20,
-            "alignment" : "left_center",
-            "wrappedButton" : 401
-        },
-
-        "reportbug":
-        {
-            "text" : "Issues",
-            "posX" : 4,
-            "posY" : 2,
+            "posY" : 26,
             "width" : 120,
             "height" : 20,
             "alignment" : "left_center",
-            "wrappedButton" : 402
+            "action" :
+            {
+                "type" : "openLink",
+                "link" : "https://github.com/OmnifactoryDevs/Omnifactory/blob/dev/UPDATENOTES.md"

```

Comment:
Would it make more sense to point it to the URL for the 1.3 tag tree? It would of course be broken until the tag exists, but it would ensure that subsequent changes to the update notes file after the release of 1.3 wouldn't show up for 1.3.

Reply:
I was thinking it would be easier to just add sections to the document, that way we do not have to re-target the link for every release, and also people could easily reference past important changes, if for some reason they had a save a couple versions old

Reply:
I just worry it could get confusing having info past the version being played on reflected there, and we also only support updating saves from one version to the next without skipping versions.

Reply:
Indeed we only support updating saves without skipping versions, but if someone has a save 2 versions old, they will need to know the information about both versions, so they can take the required steps for updating from 2 versions ago to 1 version ago, then update to the version that is 1 version old, then take the required steps for updating to the current version, and then update to the current version.

Reply:
I suppose that could work, so long as we disclaim that skipping versions is not supported and to make sure to address everything in each version's update notes section before attempting to update to the next version to avoid losing things.